app/lib/dartdoc/dartdoc_runner.dart

```diff
@@ -143,6 +149,20 @@ class DartdocJobProcessor extends JobProcessor {
       await tempDir.delete(recursive: true);
     }
 
+    if (!hasContent) {
+      suggestions.add(getDartdocRunFailedSuggestion());
+    }
+    // TODO: calculate coverage score
```

Comment:
File an issue?

Reply:
https://github.com/dart-lang/pana/issues/269 covers it.